packages/dappmanager/src/modules/stakerConfig/params.ts

```diff
@@ -0,0 +1,36 @@
+import { Network } from "@dappnode/common";
+import {
+  ExecutionClientOrSignerVersion,
+  LodestarStakersMinimumVersions
+} from "./types";
+
+const lodestarStakersPraterMinimumVersions: ExecutionClientOrSignerVersion<"prater"> =
+  {
+    "goerli-nethermind.dnp.dappnode.eth": "1.0.8",
+    "goerli-geth.dnp.dappnode.eth": "0.4.27", // Needs publishment
```

Comment:
just noticed this geth version is wrong and acutally I think we do not even need to publish a new version as [v0.4.28](https://github.com/dappnode/DAppNodePackage-goerli-geth/releases/tag/v0.4.28) already supports Lodestar

```suggestion
    "goerli-geth.dnp.dappnode.eth": "0.4.28",
```

@dsimog01 on my dappnode I am running `0.4.27` and it tells me it is the latest version, was `0.4.28` not published correctly or I am missing something?